### DISCUSSION TITLE
Forward Node in fault, no issue detected

### DISCUSSION THREAD

**Discussion by 3isenHeiM** [Security-Onion-Solutions/securityonion#9538] [Unsupported Versions]
Hello, our sensor runs perfectly fine, the `so-status` returned all green, but yet hi is marked as in Fault in the Grid overview on our Manager.

![image](https://user-images.githubusercontent.com/26417172/211353904-6c147191-e383-4a73-b076-9f24e0f6a1eb.png)

Where can I check what's wrong with it ?

**Comment by cm-ops** [comment]
On that sensor, would you run `so-status -q`, then `echo $?`, if your output is 99, reboot the node and recheck once all services are OK.

**Reply by 3isenHeiM** [reply]
Return code was indeed 99, and the reboot solved it. Thanks !
